fbw-a32nx/src/base/flybywire-aircraft-a320-neo/config/a32nx/a320-251n/README.md

```diff
@@ -0,0 +1,47 @@
+# FlyByWire Unified Config
+
+The goal of this effort is to unify configuration for all FBW based projects, in a way that can be piped to both JS and WASM Gauges,
+and in doing so, exposing these variables for both easier visibility for contributors as well as aiding external parties in adapting the
+flyPad and other components to other aircraft.
+
+Currently, there is config and hardcoded values spreadout across the repo, sometimes in various configuration (.json, .h, .toml, etc.) files.
+
+The solution to this takes on (at the moment) the form of a VFS stored .json markup files in ``config/<env.AIRCRAFT_PROJECT_PREFIX>/<Variant>/*.json`` that has a format that currently is being decided and iterated upon.
```

Comment:
Let's add json5 and make it standard - for config files json5 makes more sense as these should often be commented.